srcpkgs/python3-bibtexparser/template

```diff
@@ -1,14 +1,20 @@
 # Template file for 'python3-bibtexparser'
 pkgname=python3-bibtexparser
-version=1.1.0
-revision=5
+version=1.2.0
+revision=1
 wrksrc="bibtexparser-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-parsing python3-future"
+depends="python3-parsing"
+checkdepends="python3-future $depends"
 short_desc="Python3 package for parsing BibTeX"
 maintainer="xaltsc <xaltsc@protonmail.ch>"
 license="LGPL-3.0-only"
-homepage="https://github.com/sciunto-org/python-bibtexparser"
+homepage="https://pypi.org/project/bibtexparser/"
+changelog="https://raw.githubusercontent.com/sciunto-org/python-bibtexparser/master/CHANGELOG"
 distfiles="${PYPI_SITE}/b/bibtexparser/bibtexparser-${version}.tar.gz"
-checksum=df8966ea752db6d74657a69b9d684a61aa33457ad6d9d50e41c50ef7f374907f
+checksum=0f9ab94e3fc36ee2ee6a3713c5dd7320d4b6ee52bd66ecbab03c6b06675ae410
+
+do_check() {
+	LC_ALL="en_US.UTF-8" nosetests
```

Comment:
This should use `nosetests3` directly, but it also requires that `python3-nose` be in checkdepends.

Reply:
We can run all the tests with pytest if we just remove `do_check` and add pytest to checkdepends.